agent/grpc-internal/client.go

```diff
@@ -52,6 +52,79 @@ const grpcServiceConfig = `
 				"MaxBackoff": "5s",
 				"RetryableStatusCodes": ["RESOURCE_EXHAUSTED"]
```

Comment:
What's the difference between the policy defined in here and the one below?

Reply:
The first one makes read-and-write calls to retry only when rate-limited while the second one is defined only for read requests that are errored due to a server error.